lib/rules/no-path-concat.js

```diff
@@ -44,6 +44,14 @@ module.exports = {
 
                     context.report({ node, message: "Use path.join() or path.resolve() instead of + to create paths." });
                 }
+            },
+
+            TemplateLiteral(node) {
+                node.expressions.forEach(element => {
+                    if (element.type === "Identifier" && MATCHER.test(element.name)) {
+                        context.report({ node, message: "Use path.join() or path.resolve() instead of + to create paths." });
```

Comment:
I agree this should use a new message.

One option might be to just say "concatenation" instead of + for both messages (although I am also okay with having two different messages if you think they might be more useful/readable for users).

Reply:
concatenation is good in my opinion, I'm going to update this